### PR TITLE
[DevCenter] Preparing for azure-developer-devcenter 1.0.0b1 release

### PR DESCRIPTION
**Pull request by sebrenna**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by sebrenna** [comment]
@lmazuel AutoRestCI seems to have made a bunch of changes to the files in this PR, some of which aren't correct. (e.g. package name) Any idea what's going on?

File: sdk/devcenter/azure-developer-devcenter/sdk_packaging.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[packaging]
+package_name = "azure-developer-devcenter"
+package_nspkg = "azure-developer-nspkg"
+package_pprint_name = "MyService Management"
```
Comment (changlong-liu) [inline sdk/devcenter/azure-developer-devcenter/sdk_packaging.toml:4]: This is correct? Should be "Azure Developer DevCenter Service"?

File: sdk/devcenter/azure-developer-devcenter/README.md [hunk 2]
```diff
@@ -1,156 +1,28 @@
...
-
-```python
->>> import os
->>> from azure.developer.devcenter import DevCenterClient
->>> from azure.identity import DefaultAzureCredential
->>> tenant_id = os.environ['AZURE_TENANT_ID']
->>> client = DevCenterClient(tenant_id=tenant_id, dev_center="my_dev_center", credential=DefaultAzureCredential())
-```
-
-## Examples
```
Comment (changlong-liu) [inline sdk/devcenter/azure-developer-devcenter/README.md:47]: May I know whey these examples are removed?

Comment (chrissmiller) [reply sdk/devcenter/azure-developer-devcenter/README.md:47]: Think this is what @sebrenna is referring to -- the AutorestCI bot keeps adding a commit titled "Packaging update of azure-developer-devcenter" which breaks everything and removes our existing work. How can we disable this bot?

File: sdk/devcenter/azure-developer-devcenter/sdk_packaging.toml (hunk 1 above)
Comment (sebrenna) [reply sdk/devcenter/azure-developer-devcenter/sdk_packaging.toml:4]: @changlong-liu not our changes - this is another change that AutorestCI bot is adding to the PR. Our changes are extremely minimal and can be found in this commit: https://github.com/Azure/azure-sdk-for-python/pull/27308/commits/f41ec2172de7009991dadaaa48f7e4cbb6f036e0

Please let us know how we can disable this bot from running, or how we can get it to run properly.

Comment (sebrenna) [reply sdk/devcenter/azure-developer-devcenter/sdk_packaging.toml]: Fixed after disabling push from AutorestCI.

File: sdk/devcenter/azure-developer-devcenter/README.md (hunk 2 above)
Comment (sebrenna) [reply sdk/devcenter/azure-developer-devcenter/README.md]: Fixed after disabling push from AutorestCI.

**Comment by sebrenna** [comment]
@changlong-liu ready for review again - disabled the push from AutorestCI.

**Review by changlong-liu — APPROVED** [review]

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**